topdown/topdown_test.go

```diff
@@ -3596,3 +4128,109 @@ func init() {
 	})
 
 }
+
+func TestTopDownJWTEncodeSign(t *testing.T) {
+
+	astHeaderHS256Obj := ast.NewObject()
```

Comment:
Refactor this test case to just call ast.MustParseTerm instead of manually constructing like this. For example:

```
astHeaderHS256Obj := ast.MustParseTerm(`{"typ": "JWT", "alg": "HS256"}`)
```

Reply:
A couple of things: The current code uses this approach, the reason being that ```ast.MustParseTerm``` returns a single term irrespective of how many k,v pairs you have inside the string, therefore you can not use that to later insert in an object.  ast.NewObject() expects 2 terms representing k,v.  Do you have any other suggestion?

Reply:
Ah, I see. You shouldn't have to call `ast.NewObject(astPayloadTerm)`. If you need to access the underlying value for reason you can write:

```golang
term := ast.MustParseTerm(`{"foo": {...}}`)
obj := term.Value.(ast.Object)
```

Generally, I'd only construct AST terms dynamically if the construction itself required loops/conditionals. Since the construction is just a sequence of NewObject() -> Insert() -> ... ->  Insert(), it's more concise and clearer to simply call ast.MustParseTerm(...) and define the value as a string.

Hope this helps!

Reply:
The reason is that the builtin takes and parses objects. The objects are converted to JSON and parsed. Let me try this way to see if it works.

Reply:
Done